supervisor/const.py

```diff
@@ -382,12 +382,27 @@
 ROLE_ALL = [ROLE_DEFAULT, ROLE_HOMEASSISTANT, ROLE_BACKUP, ROLE_MANAGER, ROLE_ADMIN]
 
 
+class AddonBootConfig(StrEnum):
+    """Boot mode config for the add-on."""
+
+    AUTO = "auto"
+    MANUAL = "manual"
+    MANUAL_FORCED = "manual_forced"
```

Comment:
I find `manual_forced` rather confusing. The `boot` option says weather an add-on should be started on boot, does manual force mean that it is manually forced to always boot? :thinking: 

I found simply `disabled` better, but maybe that is just a me :sweat_smile: 

Maybe the option `boot` should actually be more verbose, like `start_on_boot`. Then `disabled` definitely would be clearer.

Maybe `disallowed`?

Reply:
Let's go with `manual_only` as discussed in our meeting.